package.json

```diff
@@ -50,6 +50,7 @@
     "@uswds/uswds": "3.1.0",
     "gulp": "^4.0.2",
     "markdown-it-attrs": "^4.1.6",
+    "net": "^1.0.2",
```

Comment:
seems to be an addition to USWDS - will investigate since I'm not too sure how it got here